cypress/e2e/default/learn/common-components/navbar.ts

```diff
@@ -0,0 +1,85 @@
+const navBarselectors = {
+  heading: "[data-test-label='landing-header']",
+  smallCallToAction: "[data-test-label='landing-small-cta']",
+  navigationLinks: '.nav-list',
+  avatarContainer: '.avatar-container',
+  defaultAvatar: '.avatar-container',
+  menuButton: '.toggle-button-nav',
+  avatarImage: '.avatar-container .avatar'
+};
+
+
+
+
+describe('Navbar when logged out', () => {
+  beforeEach(() => {
+    cy.visit('/').then(() => cy.viewport(1200, 660));
+  });
+
+  it('Should have a "Sign in" button', () => {
+    cy.contains("[data-test-label='landing-small-cta']", 'Sign in');
+  });
+
+  it(
+    'Should have `Sign in` link on landing and learn pages' +
+    ' when not signed in',
+    () => {
+      cy.contains(navBarselectors.smallCallToAction, 'Sign in');
+      cy.get(navBarselectors.menuButton).click();
+      cy.get(navBarselectors.navigationLinks).contains('Curriculum').click();
+      cy.contains(navBarselectors.smallCallToAction, 'Sign in');
+    }
+  );
+});
+
+describe('Navbar Logged in', () => {
+  beforeEach(() => {
+    cy.visit('/').then(() => cy.login()).then(() => cy.viewport(1200, 660));
```

Comment:
```suggestion
    cy.visit('/')
      .then(() => cy.login())
      .then(() => cy.viewport(1200, 660));
```

Reply:
That step was completed two days ago but I never managed to push it.